internal/align.go

```diff
@@ -1,6 +1,8 @@
 package internal
 
+import "golang.org/x/exp/constraints"
```

Comment:
A little copying is better than a little dependency; can't we declare our own

```
type Integer interface {
	~int | ~int8 | ~int16 | ~int32 | ~int64 | ~uint | ~uint8 | ~uint16 | ~uint32 | ~uint64 | ~uintptr
}
```

for now until the dust settles and package constraints is promoted out of exp? Seems overkill to pull in the whole thing for a single constraint.

Reply:
The key advantage of x/exp is this:

> If future releases of Go add new predeclared integer types, this constraint will be modified to include them.

I don't mind much either way, I do think at some point we'll probably pull it in.

Reply:
True, it's likely this will get promoted out of exp in its current shape. I've amended your commit to make exp a direct dependency in go.mod, looks like you didn't `go tidy`.